libs/language-server/src/lib/validation/checks/import-definition.ts

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import { type ImportDefinition } from '../../ast/generated/ast';
+import { type JayveeValidationProps } from '../validation-registry';
+
+export function validateImportDefinition(
+  importDefinition: ImportDefinition,
+  props: JayveeValidationProps,
+): void {
+  checkPathExists(importDefinition, props);
+}
+
+function checkPathExists(
+  importDefinition: ImportDefinition,
+  props: JayveeValidationProps,
+): void {
+  const resolvedImport = props.importResolver.resolveImport(importDefinition);
+  if (resolvedImport === undefined) {
+    props.validationContext.accept('error', 'Import cannot be resolved.', {
```

Comment:
Would be stronger as `Import from "{path}" could not be resolved. File does not exist or is not readable.` (if that is actually true / possible). Having the path in there will be very helpful for future debugging.

Reply:
Made the error message a little more vague by saying "Check if the file exists".
Tackled in [63c607b](https://github.com/jvalue/jayvee/pull/564/commits/63c607b9370100465427fe2b5c3cd47c90612211)